### ISSUE TITLE
/etc/init.d/zfs-mount refers to 'zfs' instead of 'zpool'

### ISSUE THREAD

**Issue by mailinglists35** [zfsonlinux/pkg-zfs#121]
from /etc/init.d/zfs-mount on my debian/wheezy 0.6.3 debian-zfs

```
# Short-Description: Import and mount ZFS pools, filesystems and volumes
# Description: Run the `zfs import`, `zfs mount -a`, `zfs umount -a` or
#              `zfs export` command.

I believe the correct wording should instead be this:

# Description: Run the `zpool import`, `zfs mount -a`, `zfs umount -a` or
#              `zpool export` command.
```

<b>EDIT:</b> @behlendorf updated markdown for readability.

**Comment by FransUrbo** [comment]
> Description: Run the zfs import, zfs mount -a, zfs umount -a or zfs export command.
> 
> I believe the correct wording should instead be this:
> 
> Description: Run the zpool import, zfs mount -a, zfs umount -a or zpool export command."
> 
> You're of course right! I'll deal with this at the next upload (if I remember :).

Thanx for letting us know...

**Comment by FransUrbo** [comment]
Fixed in <code>0.6.3-13~cd3939~wheezy</code>.
